userguide/intro.rst

```diff
@@ -357,7 +357,7 @@ U2
 * The :guilabel:`Endpoint does not support regions` and
```

Comment:
I thought we changed that, please verify field label is correct here.

Reply:
Rechecked: it is correct for the legacy UI